RecoTracker/DeDx/plugins/HLTDeDxFilter.cc

```diff
@@ -128,7 +130,7 @@ bool HLTDeDxFilter::hltFilter(edm::Event& iEvent,
   }
   for (unsigned int i = 0; i < trackCollection.size(); i++) {
     reco::TrackRef track = reco::TrackRef(trackCollectionHandle, i);
-    if (pt[i] > minPT_ && fabs(eta[i]) < maxETA_ && dEdxTrack[track].numberOfMeasurements() > minNOM_ &&
+    if (pt[i] > minPT_ && fabs(eta[i]) >= minETA_ && fabs(eta[i]) < maxETA_ && dEdxTrack[track].numberOfMeasurements() > minNOM_ &&
```

Comment:
@fmanteca I know it was like that before, but would you mind changing `fabs` with `std::abs` in both places in this expression?

Reply:
done